logbook-core/src/main/java/org/zalando/logbook/DefaultHttpLogFormatter.java

```diff
@@ -42,7 +43,12 @@ public String format(final Precorrelation<HttpRequest> precorrelation) throws IO
     }
 
     private String formatRequestLine(final HttpRequest request) {
-        return String.format("%s %s HTTP/1.1", request.getMethod(), request.getRequestUri());
+        return String.format("%s %s HTTP/1.1", request.getMethod(), renderRequestUri(request));
+    }
+
+    private String renderRequestUri(HttpRequest request) {
+        final String query = Joiner.on("&").join(request.getQueryParameters().entries());
```

Comment:
This looks like it is relying on the `toString` implementation of a `Map.Entry`, I don't think this is guaranteed to be in the wanted format.